recipes-core/init-ifupdown/init-ifupdown/ccimx6ulsbcpro/interfaces

```diff
@@ -0,0 +1,46 @@
+# /etc/network/interfaces -- configuration file for ifup(8), ifdown(8)
```

Comment:
The addition of the interfaces enabling those devices will require that all users have all infrastructure to wifi installed on every image, even `core-image-minimal` otherwise the board will lock during the boot.

Is this the intended behavior?

Reply:
Hi Otavio, if there is no wifi infrastructure there will be no wlann interfaces so what we see for example with core-image-minimal is:
`Mon Mar  4 09:53:33 2019: Configuring network interfaces... ifconfig: SIOCGIFFLAGS: No such device
Mon Mar  4 09:53:33 2019: ifconfig: SIOCSIFADDR: No such device
Mon Mar  4 09:53:33 2019: ifconfig: SIOCGIFFLAGS: No such device
Mon Mar  4 09:53:33 2019: ifconfig: SIOCSIFADDR: No such device
Mon Mar  4 09:53:33 2019: udhcpc: started, v1.30.1
Mon Mar  4 09:53:33 2019: udhcpc: sending discover
Mon Mar  4 09:53:36 2019: udhcpc: sending discover
Mon Mar  4 09:53:37 2019: udhcpc: sending select for 192.168.3.214
Mon Mar  4 09:53:37 2019: udhcpc: lease of 192.168.3.214 obtained, lease time 14592
Mon Mar  4 09:53:37 2019: /etc/udhcpc.d/50default: Adding DNS 10.49.8.62
Mon Mar  4 09:53:37 2019: /etc/udhcpc.d/50default: Adding DNS 10.10.8.62
`
Anyway, I am still working on a PR to add the wireless changes, so I could remove the wireless interfaces from this PR.

Could you maybe extend on why did you say the board would lock up during boot?

Thanks,

Reply:
I have removed the wireless interfaces from the interfaces file for the time being. I will add them with the bulk of wireless changes.